rust/cedar-policy-mcp-schema-generator/src/cli/args.rs

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Cedar Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use clap::{Args, Parser, Subcommand, ValueEnum};
+use serde::Serialize;
+use std::path::PathBuf;
+
+#[derive(Args, Clone, Debug, Serialize)]
+#[clap(next_help_heading = "Configuration Options")]
+#[serde(rename_all = "kebab-case")]
+pub(crate) struct ConfigOptions {
+    /// Whether to encode the `OutputSchema` of each tool as an optional attribute of the tool's action's context (default: false).
+    #[arg(long, default_value_t = false)]
+    pub(crate) include_outputs: bool,
+    /// Whether to encode "object" typed properties as a Record Type when the object does not have `additionalProperties` (default: false).
+    #[arg(long, default_value_t = false)]
+    pub(crate) objects_as_records: bool,
+    /// Whether to keep `mcp_principal`, `mcp_resource`, `mcp_context`, and `mcp_action` annotations in the final schema (default: false).
+    #[arg(long, default_value_t = true)]
+    pub(crate) keep_annotations: bool,
+    /// Whether to create an output schema with a single namespace by flattening names---e.g., `Foo::Baz::Bar` becomes `Foo_Baz_Bar`---(default: false).
+    #[arg(long, default_value_t = false)]
+    pub(crate) flatten_namespaces: bool,
+}
+
+#[derive(ValueEnum, Clone, Copy, Debug, Serialize)]
+pub(crate) enum OutputFormat {
+    /// Human Readable Cedar Schema Format.
+    Human,
+    /// Json Cedar Schema Format.
+    Json,
+}
+
+#[derive(ValueEnum, Clone, Copy, Debug, Serialize)]
+pub enum ErrorFormat {
+    /// Human-readable error messages with terminal graphics and inline code snippets.
+    Human,
+    /// Plain-text error messages without fancy graphics or colors, suitable for screen readers.
+    Plain,
+    /// Machine-readable JSON output.
+    Json,
+}
+
+#[derive(Clone, Debug, Serialize, Subcommand)]
+pub(crate) enum Command {
+    /// Generate a Cedar Schema by adding an action for each tool description of an MCP
```

Comment:
Might need the period depending on how this gets rendered by clap

```suggestion
    /// Generate a Cedar Schema by adding an action for each tool description of an MCP.
```